homeassistant/components/gree/switch.py

```diff
@@ -26,6 +26,7 @@ def init_device(coordinator):
         async_add_entities(
             [
                 GreePanelLightSwitchEntity(coordinator),
+                GreeHealthModeSwitchEntity(coordinator),
```

Comment:
I guess this means all gree devices will get a health mode switch, regardless of if they support it or not.
Is there any way to check if the device supports health mode?

Reply:
According to https://github.com/cmroche/greeclimate/issues/5 there isn't.

Many (most) units don't support Fresh Air feature either, so I thought it wouldn't be a problem, plus from my testing it doesn't affect anything, unsupported entities can be simply disabled.

If you tell me how to disable an entity from the start, I can do it, but mt knowledge with python and HA is limited.

Reply:
Just add `_attr_entity_registry_enabled_default = False` to the class definition, and the entity will be disabled.
Maybe that should be done also for other features which are rarely supported?

Reply:
To be honest, I'm not sure what can be considered as a reasonable minimum featureset that can be enabled.

I'll make health mode start off as disabled.

BTW my ACs came with a remote that supports multiple features my AC units don't, so I'm guessing this is a Gree thing.

Reply:
@emontnemery Check out my changes, I've made health mode not load by default + updated tests.